lib/Instance.js

```diff
@@ -482,6 +482,14 @@ function Instance(Model, opts) {
 	var addInstanceProperty = function (key) {
 		var defaultValue = null;
 		var prop = Model.allProperties[key];
+		var isEnumerable;
+
+		// Defaults to true unless explicitly set to false
+		isEnumerable = !(
+			prop && prop.hasOwnProperty &&
+			prop.hasOwnProperty('enumerable') &&
+			!prop.enumerable
+		);
```

Comment:
Nice, I'll move this code this week. I'll also add some tests (I tried to run them on my macbook but had some issues. I'll investigate further...)